test/ActiveLogin.Authentication.BankId.AspNetCore.Test/BankId_Authentication_Tests.cs

```diff
@@ -177,6 +177,7 @@ public async Task AutoLaunch_Sets_Correct_RedirectUri()
 	        // Arrange mocks
 	        var autoLaunchOptions = new BankIdLoginOptions(new List<string>(), null, false, true, false, false);
 	        var mockProtector =  new Mock<IBankIdLoginOptionsProtector>();
+            var mockQrCodeGenerator = new Mock<IBankIdQrCodeGenerator>();
```

Comment:
Technically, we don't need this anymore. right?

Reply:
Nope!